config.gemspec

```diff
@@ -28,7 +28,7 @@ Please consider donating to our open collective to help us maintain this project
   s.required_ruby_version = '>= 2.4.0'
 
   s.add_dependency 'deep_merge', '~> 1.2', '>= 1.2.1'
-  s.add_dependency 'dry-validation', '~> 1.0'
+  s.add_dependency 'dry-validation', '~> 1.4.0'
```

Comment:
Why are you pinning so tightly here? A comment would be helpful.

Reply:
Do you think `s.add_dependency 'dry-validation', '~> 1.4'` would be better?

Reply:
Yes, much better. Thanks